### DISCUSSION TITLE
V - Rising

### DISCUSSION THREAD

**Discussion by ScouserBird** [gameserverapp/Platform#176] [Suggestions]
Currently Beta Testing this game so unable to give any information other than what is already publicly out there by the Devs.

https://playvrising.com/

All I will say is that this could possibly be as much of a hit as Valheim! Maybe worth considering when it is released into EA this month.

**Comment by Eurymachus** [comment]
For sure! This game has massive potential and is quite popular already.

**Comment by Sarpendon** [comment]
yes please would be awesome
